kolibri/core/assets/src/api-resource.js

```diff
@@ -322,6 +322,23 @@ class Collection {
 
   set synced(value) {
     this._synced = value;
+    this.models.forEach((model) => { model.synced = true; });
+  }
+
+  static key(params) {
+    // Sort keys in order, then assign those keys to an empty object in that order.
+    // Then stringify to create a cache key.
+    return JSON.stringify(
+      Object.assign(
+        {}, ...Object.keys(params).sort().map(
+          (paramKey) => ({ [paramKey]: params[paramKey] })
+        )
+      )
+    );
+  }
+
+  get key() {
```

Comment:
To clear up my understanding, when we ask for `collection.key` that calls this getter method, which in turn calls the static method to produce the key?